__tests__/server/config/env/runTime.spec.js

```diff
@@ -125,6 +127,36 @@ describe('runTime', () => {
     });
   });
 
+  describe('ONE_DANGEROUSLY_DISABLE_CSP', () => {
+    const disableCspEnv = getEnvVarConfig('ONE_DANGEROUSLY_DISABLE_CSP');
+
+    it('throws error if ONE_DANGEROUSLY_DISABLE_CSP is set to true and NODE_ENV is not in development', () => {
+      expect(() => disableCspEnv.validate('true')).toThrowError('If you are trying to bypass CSP requirement, NODE_ENV must also be set to development.');
+    });
+
+    it('warns console if both ONE_DANGEROUSLY_DISABLE_CSP and NODE_ENV are set properly', () => {
+      process.env.NODE_ENV = 'development';
```

Comment:
This never gets reset, and the `beforeEach` isn't there, was it removed ?

Reply:
this was never in before each it just has a call to reset the NODE_ENV env var

Reply:
resetEnvVar('NODE_ENV'); <- this will delete that env var

Reply:
Where is this called ?

Reply:
```js
beforeEach(() => {
    // eslint-disable-next-line no-console
    console.info = jest.fn();
    console.warn = jest.fn();
    resetEnvVar('NODE_ENV');
    resetEnvVar('ONE_DANGEROUSLY_DISABLE_CSP', 'false');
    resetEnvVar('HTTP_ONE_APP_DEV_CDN_PORT');
    jest.resetModules();
    jest.resetAllMocks();
  });```